### ISSUE TITLE
Could not find command "readme"

### ISSUE THREAD

**Issue by n-rodriguez** [rubygems/gemstash#316]
```sh
nonroot@a8f1580e8ac9:/app$ bin/gemstash  
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
Could not find command "readme".
```

**Comment by n-rodriguez** [comment]
Because there's nothing there : https://github.com/rubygems/gemstash/blob/master/gemstash.gemspec#L26

**Comment by olleolleolle** [comment]
@n-rodriguez You seem to be correct in that adding the lib/gemstash/man directory does nothing: we could remove that from the gemspec.

**Comment by n-rodriguez** [comment]
> Now I recall, there are more steps to making the documentation (rake tasks to run on releasing).

yeah : https://github.com/rubygems/gemstash/blob/master/Rakefile#L24

**Comment by n-rodriguez** [comment]
It is fixed with `gemstash 2.1.0` :+1: 

```sh
nonroot@9f0392c3b55c:/app$ bin/gemstash 
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
gemstash-readme(7)					    gemstash-readme(7)



1mGEMSTASH0m
   1mWHAT IS GEMSTASH?0m
       Gemstash	   is	 both	a   cache   for	  remote   servers   such   as
       https://rubygems.org, and a private gem source.

       If you are using bundler (http://bundler.io/) across many machines that
       have access to a server within your control, you might want to use Gem-
       stash.

       If you produce gems that you don't want everyone in the world  to  have
       access to, you might want to use Gemstash.

       If you frequently bundle the same set of gems across multiple projects,
       you might want to use Gemstash.

       Are you only using gems from https://rubygems.org, and don't bundle the
       same gems frequently?  Well, maybe you don't need Gemstash... yet.

       Gemstash	 is maintained by Ruby Together (https://rubytogether.org/), a
       grassroots initiative committed to supporting the critical Ruby	infra-
       structure   you	 rely	on.    Contribute   today   as	an  individual
       (https://rubytogether.org/developers) or	 even  better,	as  a  company
       (https://rubytogether.org/companies),	and   ensure   that   Bundler,
       RubyGems, Gemstash, and other shared tooling is	around	for  years  to
       come.

   1mQUICKSTART GUIDE0m
   1mSETUP0m
       Gemstash	 is  designed  to be quick and painless to get set up.	By the
       end of this Quickstart Guide, you will be able to bundle	 stashed  gems
       from public sources against a Gemstash server running on your machine.

       Install Gemstash to get started:

	      $ gem install gemstash

       After  it is installed, starting Gemstash requires no additional steps.
       Simply start the Gemstash server with the gemstash command:

	      $ gemstash start

       You may have noticed that the command finished quickly.	 This  is  be-
       cause  Gemstash	will run the server in the background by default.  The
       server runs on port 9292.

   1mBUNDLING0m
       With the server running, you can bundle against it.  Tell Bundler  that
       you want to use Gemstash to find gems from RubyGems.org:

	      $ bundle config mirror.https://rubygems.org http://localhost:9292

       Now you can create a Gemfile and install gems through Gemstash:

	      # ./Gemfile
	      source "https://rubygems.org"
	      gem "rubywarrior"

       The  gems you include should be gems you don't yet have installed, oth-
       erwise Gemstash will have nothing to stash.  Now bundle:

	      $ bundle install --path .bundle

       Your Gemstash server has fetched the gems from https://rubygems.org and
       cached  them for you! To prove this, you can disable your Internet con-
       nection and try again.  Gem files (*.gem) are cached indefinitely.  Gem
       dependencies metadata are cached for 30 minutes, so if you bundle again
       before that, you can successfully bundle without	 an  Internet  connec-
       tion:

	      $ # Disable your Internet first!
	      $ rm -rf Gemfile.lock .bundle
	      $ bundle

   1mFALLING BACK TO RUBYGEMS.ORG0m
       If  you	want to make sure that your bundling from https://rubygems.org
       still works as expected when the Gemstash server is  not	 running,  you
       can easily configure Bundler to fallback to https://rubygems.org.

	      $ bundle config mirror.https://rubygems.org.fallback_timeout true

       You can also configure this fallback as a number of seconds in case the
       Gemstash server is simply unresponsive.	This example uses a  3	second
       timeout:

	      $ bundle config mirror.https://rubygems.org.fallback_timeout 3

   1mSTOPPING THE SERVER0m
       Once  you've finish using your Gemstash server, you can stop it just as
       easily as you started it:

	      $ gemstash stop

       You'll also want to tell Bundler that it can go back  to	 getting  gems
       from RubyGems.org directly, instead of going through Gemstash:

	      $ bundle config --delete mirror.https://rubygems.org

   1mUNDER THE HOOD0m
       You might wonder where the gems are stored.  After running the commands
       above, you will find a new directory at	~/.gemstash.   This  directory
       holds  all  the cached and private gems.	 It also has a server log, the
       database, and configuration for Gemstash.  If you prefer, you can point
       to a different directory (gemstash help customize.7).

       Gemstash	 uses  SQLite (https://www.sqlite.org/) to store details about
       private gems.  The database will be located in ~/.gemstash, however you
       won't  see  the database appear until you start using private gems.  If
       you prefer, you can use a different  database  (gemstash	 help  custom-
       ize.7).

       Gemstash	 temporarily  caches  things  like gem dependencies in memory.
       Anything cached in memory will last for 30  minutes  before  being  re-
       trieved	again.	 You can use memcached (gemstash help customize.7) in-
       stead of caching in memory.  Gem files are always  cached  permanently,
       so  bundling  with  a Gemfile.lock with all gems cached will never call
       out to https://rubygems.org.

       The server you ran is provided  via  Puma  (http://puma.io/)  and  Rack
       (http://rack.github.io/),  however  they	 are  not customizable at this
       point.

   1mDEEP DIVE0m
       Deep dive into more subjects:

       o Private gems (gemstash help private-gems.7)

       o Multiple gem sources (gemstash help multiple-sources.7)

       o Using Gemstash as a mirror (gemstash help mirror.7)

       o Customizing the server (database, storage, caching, and  more)	 (gem-
	 stash help customize.7)

       o Deploying Gemstash (gemstash help deploy.7)

       o Debugging Gemstash (gemstash help debugging.7)

   1mREFERENCE0m
       An anatomy of various configuration and commands:

       o Configuration (gemstash help configuration.5)

       o Authorize (gemstash help authorize.1)

       o Start (gemstash help start.1)

       o Stop (gemstash help stop.1)

       o Status (gemstash help status.1)

       o Setup (gemstash help setup.1)

       o Version (gemstash help version.1)

       To  see	what  has  changed  in	recent	versions  of Gemstash, see the
       CHANGELOG    (https://github.com/rubygems/gemstash/blob/master/CHANGEL-
       OG.md).

   1mDEVELOPMENT0m
       After  checking	out  the  repo, run bin/setup to install dependencies.
       Then, run rake to run RuboCop and the tests.  While developing, you can
       run  bin/gemstash to run Gemstash.  You can also run bin/console for an
       interactive prompt that will allow you to experiment.

   1mCONTRIBUTING0m
       Bug  reports   and   pull   requests   are   welcome   on   GitHub   at
       https://github.com/rubygems/gemstash.  This project is intended to be a
       safe, welcoming space for collaboration, and contributors are  expected
       to adhere to the Contributor Covenant (https://github.com/rubygems/gem-
       stash/blob/master/CODE_OF_CONDUCT.md) code of conduct.

   1mLICENSE0m
       The gem is available as open source under the terms of the MIT  License
       (http://opensource.org/licenses/MIT).



			       November 30, 2015	    gemstash-readme(7)
```

**Comment by n-rodriguez** [comment]
@olleolleolle thank you!
